src/physicsobjects.hxx

```diff
@@ -381,36 +381,60 @@ auto PtCorrection_byValue(auto &df, const std::string &corrected_pt,
 /// energy correction
```

Comment:
Add a link to the crosspog overview here: 

https://cms-nanoaod-integration.web.cern.ch/commonJSONSFs/TAU_tau_Run2_UL/TAU_tau_2018_UL.html

and for the other eras as well, as a quick reference